darts/models/forecasting/sf_models.py

```diff
@@ -0,0 +1,357 @@
+        <https://github.com/Nixtla/statsforecast>`_.
+
+        Automatically selects the best Theta (Standard Theta Model (‘STM’), Optimized Theta Model (‘OTM’),
+        Dynamic Standard Theta Model (‘DSTM’), Dynamic Optimized Theta Model (‘DOTM’)) model using mse.
+        <https://www.sciencedirect.com/science/article/pii/S0169207016300243>
+
+        It is probabilistic, whereas :class:`FourTheta` is not.
+
+        We refer to the `statsforecast AutoTheta documentation
+        <https://nixtla.github.io/statsforecast/models.html#autotheta>`_
+        for the documentation of the arguments.
+
+        Parameters
+        ----------
+        autotheta_args
+            Positional arguments for ``statsforecasts.models.AutoTheta``.
+        autotheta_kwargs
+            Keyword arguments for ``statsforecasts.models.AutoTheta``.
+
+            ..
+
+        Examples
+        --------
+        >>> from darts.models import StatsForecastAutoTheta
+        >>> from darts.datasets import AirPassengersDataset
+        >>> series = AirPassengersDataset().load()
+        >>> model = StatsForecastAutoTheta(season_length=12)
+        >>> model.fit(series[:-36])
+        >>> pred = model.predict(36, num_samples=100)
+        """
+        super().__init__()
+        self.model = SFAutoTheta(*autotheta_args, **autotheta_kwargs)
+
+    def __str__(self):
+        return "Auto-Theta-Statsforecasts"
+
+    def fit(self, series: TimeSeries):
+        super().fit(series)
+        self._assert_univariate(series)
+        series = self.training_series
+        self.model.fit(
+            series.values(copy=False).flatten(),
+        )
+        return self
+
+    def predict(
+        self,
+        n: int,
+        num_samples: int = 1,
+        verbose: bool = False,
+    ):
+        super().predict(n, num_samples)
+        forecast_dict = self.model.predict(
+            h=n,
+            level=(68.27,),  # ask one std for the confidence interval.
+        )
+
+        mu, std = unpack_sf_dict(forecast_dict)
+        if num_samples > 1:
+            samples = create_normal_samples(mu, std, num_samples, n)
+        else:
+            samples = mu
+
+        return self._build_forecast_series(samples)
+
+    @property
+    def min_train_series_length(self) -> int:
+        return 10
+
+    def _supports_range_index(self) -> bool:
+        return True
+
+    def _is_probabilistic(self) -> bool:
+        return True
+
+
+def create_normal_samples(
+    mu: float,
+    std: float,
+    num_samples: int,
+    n: int,
+) -> np.array:
+    """Generate samples assuming a Normal distribution."""
+    samples = np.random.normal(loc=mu, scale=std, size=(num_samples, n)).T
+    samples = np.expand_dims(samples, axis=1)
+    return samples
+
+
+def unpack_sf_dict(
+    forecast_dict: dict,
+):
+    """Unpack the dictionary that is returned by the StatsForecast 'predict()' method."""
+    mu = forecast_dict["mean"]
+    std = forecast_dict["hi-68.27"] - mu
```

Comment:
You could perhaps have the value `68.27` defined as a constant at the top of the file and reused/imported in the different models, so we avoid any risk of copying it wrong as it's used in different places.